.docker/wordpress/Dockerfile

```diff
@@ -1,15 +1,18 @@
-FROM wordpress:5.9-php8.0-fpm
+FROM wordpress:6.4-php8.3-fpm
 
-# Install xdebug
-RUN pecl install xdebug-3.1.4 \
-    && docker-php-ext-enable xdebug
+# Install PHP Extensions
+ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
+RUN chmod uga+x /usr/local/bin/install-php-extensions && sync \
+    && install-php-extensions \
+    xdebug \
+    intl
 
-# Install intl
+# System Update and libicu-dev Installation
 RUN apt-get update \
-    && apt-get install -y libicu-dev \
-    && docker-php-ext-install intl
+    && apt-get install -y \
+      libicu-dev
```

Comment:
Is this necessary?

Reply:
I don't know yet. At the actual script it's installed and I just maintain it.

Actual source: https://github.com/LibreCodeCoop/wordpress-docker/blob/main/.docker/wordpress/Dockerfile#L9

Reply:
I think that we can remove because if is necessary, the `docker-php-extension-installer` solve this dependency.

Reply:
I'll make this modification.